sirepo/server.py

```diff
@@ -427,34 +428,24 @@ def api_saveSimulationData():
     return api_simulationData(
         d.simulationType,
         d.models.simulation.simulationId,
-        pretty=False,
     )
 
 
 @api_perm.require_user
-def api_simulationData(simulation_type, simulation_id, pretty, section=None):
+def api_simulationData(simulation_type, simulation_id, pretty=False, section=None):
     """First entry point for a simulation
 
     Might be non-session simulation copy (see `simulation_db.CopyRedirect`).
     We have to allow a non-user to get data.
     """
+    #TODO(pjm): pretty is an unused argument
```

Comment:
Do we ever want pretty in this case? If not, we should create another issue that removes pretty and fixes tests.

Reply:
Several APIs have unused args at this point, downloadFile, listFiles and simulationData. Removing arguments from the API in the past caused really bad behavior like client loops in the past. Removing the args needs to be done carefully, or the API should get renamed at the same time.